lib/decompress.ps1

```diff
@@ -401,8 +391,7 @@ function Expand-WixInstaller {
         [Switch]
         $Removal
     )
-    $null = ensure "$DestinationPath\_tmp"
-    Expand-DarkArchive -Path $Path -DestinationPath "$DestinationPath\_tmp" -Removal:$Removal
+    Expand-DarkArchive -Path $Path -DestinationPath (ensure "$DestinationPath\_tmp") -Removal:$Removal
```

Comment:
I do not know how will this behave. 

THis comment is for myself to remind me to check :D

Reply:
`ensure` return `Resolve-Path $dir`, so it musts to be `Out-Null`ed or can be used here :)

Reply:
Thank you.
I was not sure about whole output of this command.

Reply:
I'm not sure why it is designed to output path... All usages that I can see are piped with `Out-Null` :smile:

Reply:
It is abused as shortcut to:

```
if ( !Test-Path $path) {
    New-Item -Folder -Type Directory
}
```

Reply:
Rewriting with `-PassThru` should be much better.